src/styles/categories.scss

```diff
@@ -0,0 +1,65 @@
+.categories {
+  padding-top: 96px;
+  @include contentPaddingInline;
+
+  @include onTablet {
+    padding-top: 120px;
+  }
+
+  &__wrapper {
+    display: grid;
+    row-gap: 40px;
+
+    @include onTablet {
+      row-gap: 64px;
+    }
+  }
+}
+
+.category {
+  &__photos {
+    @include pageGrid;
+    margin-bottom: 16px;
+  }
+
+  &__link {
+    &--wide {
+      @include onTablet {
+        grid-column: span 4;
+      }
+
+      @include onDesktop {
+        grid-column: span 8;
+      }
+    }
+
+    &--square {
+      @include onTablet {
+        grid-column: span 2;
+      }
+
+      @include onDesktop {
+        grid-column: span 4;
+      }
+    }
+  }
+
+  &__photo {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    transition: all 0.3s;
```

Comment:
Checklist p.9 The speed of animations should be the same throughout the page. There is another speed in file button.scss.